nautobot/cloud/filters.py

```diff
@@ -59,5 +64,50 @@ class CloudTypeFilterSet(NautobotFilterSet):
     content_types = ContentTypeMultipleChoiceFilter(choices=(("cloud", "cloudaccount"),))
 
     class Meta:
-        model = CloudType
-        fields = ["id", "name", "description"]
+        model = models.CloudType
+        fields = ["id", "name", "description", "tags"]
+
+
+class CloudNetworkFilterSet(NautobotFilterSet):
+    q = SearchFilter(
+        filter_predicates={
+            "name": "icontains",
+            "description": "icontains",
+        },
+    )
+    cloud_type = NaturalKeyOrPKMultipleChoiceFilter(
+        queryset=models.CloudType.objects.all(),
+        label="Cloud type (name or ID)",
+    )
+    cloud_account = NaturalKeyOrPKMultipleChoiceFilter(
+        queryset=models.CloudAccount.objects.all(),
+        label="Cloud account (name or ID)",
+    )
+    parent = NaturalKeyOrPKMultipleChoiceFilter(
+        queryset=models.CloudNetwork.objects.all(),
+        label="Parent cloud network (name or ID)",
+    )
+
+    class Meta:
+        model = models.CloudNetwork
+        fields = ["id", "name", "description", "tags"]
+
+
+class CloudNetworkPrefixAssignmentFilterSet(BaseFilterSet):
+    q = SearchFilter(
+        filter_predicates={
+            "cloud_network__name": "icontains",
+            "cloud_network__description": "icontains",
```

Comment:
Seems like most of our models include `description` in their search... is that your concern?

Reply:
Or were you wondering if this model needs a search filter at all?

Reply:
Yeah I was wondering if this model needs a search filter since it is an intermediate model and does not have a list view. On second thought, we still do need it in the RestAPI. We can keep it as it is.